src/Symfony/Component/HttpKernel/HttpKernel.php

```diff
@@ -154,18 +154,17 @@ private function handleRaw(Request $request, int $type = self::MASTER_REQUEST)
             $event = new GetResponseForControllerResultEvent($this, $request, $type, $response);
             $this->dispatcher->dispatch(KernelEvents::VIEW, $event);
 
-            if ($event->hasResponse()) {
-                $response = $event->getResponse();
-            } else {
-                $msg = sprintf('The controller must return a "Symfony\Component\HttpFoundation\Response" object but it returned %s.', $this->varToString($response));
-
-                // the user may have forgotten to return something
-                if (null === $response) {
-                    $msg .= ' Did you forget to add a return statement somewhere in your controller?';
-                }
-
-                throw new ControllerDoesNotReturnResponseException($msg, $controller, __FILE__, __LINE__ - 17);
+            if (!$event->hasResponse()) {
+                $message = sprintf(
```

Comment:
please revert unrelated changes on this line

Reply:
done ☝